src/backend/distributed/planner/function_call_delegation.c

```diff
@@ -526,7 +526,10 @@ ShardPlacementForFunctionColocatedWithDistTable(DistObjectCacheEntry *procedure,
 		if (partitionParam->paramkind == PARAM_EXTERN)
 		{
 			/* Don't log a message, we should end up here again without a parameter */
-			DissuadePlannerFromUsingPlan(plan);
+			if (plan)
```

Comment:
I think it's worth updating the comment that sometimes there is no plan at this stage at all yet, for instance when a `CALL` statement is prepared.

Reply:
yep, applied.